CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/MainActivity.kt

```diff
@@ -48,7 +50,7 @@ class MainActivity : AppCompatActivity() {
         // Before setting full screen flags, we must wait a bit to let UI settle; otherwise, we may
         // be trying to set app to immersive mode before it's ready and the flags do not stick
         container.postDelayed({
-            container.systemUiVisibility = FLAGS_FULLSCREEN
```

Comment:
It's unfortunate that `FLAGS_FULLSCREEN` can't be removed entirely, it looks like `showImmersive()` still uses it for the `AlertDialog`.

I think it should be possible to update that behavior too, as well as updating the display cutout logic as well.

Reply:
yeah, will have a good look at this one and get it migrated next.  add it into this bug https://github.com/android/camera-samples/issues/393 to address in the next change.